packages/EasyTest/src/Bridge/PhpUnit/Extension/MailerMessageLoggerListenerPHPUnitExtension.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyTest\Bridge\PhpUnit\Extension;
+
+use EonX\EasyTest\Bridge\Symfony\Mailer\EventListener\MailerMessageLoggerListenerStub;
+use PHPUnit\Runner\AfterTestHook;
+
+final class MailerMessageLoggerListenerPHPUnitExtension implements AfterTestHook
```

Comment:
Usually we don't write several uppercase characters in class names (`...PHPUnit...` in this case). More than that you have named the namespace by applying this rule (`PhpUnit`) :-)

Reply:
Done